src/svg2pdf.js

```diff
@@ -1209,24 +1248,60 @@ SOFTWARE.
   // draws the element referenced by a use node, makes use of pdf's XObjects/FormObjects so nodes are only written once
   // to the pdf document. This highly reduces the file size and computation time.
   var use = function (node, tfMatrix, refsHandler) {
+    var pf = parseFloat;
```

Comment:
there are a lot of duplicate var declarations in this method. Is probably better to declare them once at the start.